modules/compute/eks/stack.py

```diff
@@ -477,6 +482,16 @@ def _create_eks_cluster(
         Creates an Amazon EKS cluster with the specified configuration.
         """
 
+        if eks_compute_config.get("eks_api_endpoint_private"):
+            api_endpoint = eks.EndpointAccess.PRIVATE
+        else:
+            api_endpoint = eks.EndpointAccess.PUBLIC_AND_PRIVATE
+            if eks_compute_config.get("ips_to_whitelist"):
+                cidrs_from_ssm = self._fetch_cidrs_from_ssm(json.loads(eks_compute_config.get("ips_to_whitelist")))
```

Comment:
I'm not sure I understand the logic.  If an SSM path is passed in get the cidrs, else NONE.  Also, fetch cidrs from the control plane. Merge them and apply.

What if customer wants to pass in a dedicated CIDR range in their VPC?  I may be misunderstanding here...so take with a grain of salt

Reply:
As discussed and clarified offline, resolving the thread here